docs/guides/admin/admin-cli.md

```diff
@@ -462,7 +462,19 @@ tsctl similarity_score
 
 #### sketch-info Get information about a sketch
 
-Displays various information about a given sketch.
+Displays detailed information about a specific sketch.
+
+This command retrieves and displays comprehensive information about a
+Timesketch sketch, including:
+
+- **Sketch Details:** The sketch's ID and name.
+- **Active Timelines:** A table listing the active timelines within the
```

Comment:
What about timelines in other status? I would expect to see all not just the active ones.